src/transformers/feature_extraction_utils.py

```diff
@@ -480,6 +482,10 @@ def to_json_string(self) -> str:
         for key, value in dictionary.items():
             if isinstance(value, np.ndarray):
                 dictionary[key] = value.tolist()
+            elif key == "_processor_class" and value is not None:
+                # make sure private name "_processor_class" is correctly
+                # saved as "processor_class"
```

Comment:
This comment fits in one line.
(Are you nudging me toward writing a comment-styler now? ;-) )

Reply:
haha good catch!